src/extension.ts

```diff
@@ -87,7 +87,10 @@ function onActivate(context: vscode.ExtensionContext) {
 
       const execute = (o:any) => {
         graphvizView.revealOrCreatePreview(
-          vscode.ViewColumn.Beside,
+          {
+            viewColumn: vscode.ViewColumn.Beside,
+            preserveFocus: settings.extensionConfig().get("preserveFocus"),
```

Comment:
Thanks for this PR, esp. for making `preserveFocus: true` the default. It solves a lot of problems when using the extension programmatically from other extensions.
But it'd have been better to allow it to be configurable via the command options. Otherwise behavior is dependent on the user settings.

Reply:
I created #131 for this.